### PR TITLE
ON-7: Add Property Role

### PR DESCRIPTION
**Pull request by KKucht**

### CODE REVIEW

File: ontology_api/role/role_service.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        code = f"""
+if FunctionalProperty in onto_property.is_a:
+    src_instance.{model_in.role_name} = dst_instance
+else:
+    if src_instance.{model_in.role_name} is not None:
+        src_instance.{model_in.role_name}.append(dst_instance)
+    else:
+        src_instance.{model_in.role_name} = [dst_instance]                
+"""
+        exec(code)
```
Comment (mrwrob) [inline ontology_api/role/role_service.py:53]: I don't like this code at all. 
What is the `src_instance`? Is it a class? A dict? If it's a class you can probably use `getattr`... 
`exec` is almost never a good idea...

**Review by mrwrob — CHANGES_REQUESTED** [review]
Please refactor `RoleService` class to remove `exec` call

**Review by PLangowski — APPROVED** [review]
Nice!

**Comment by mrwrob** [comment]
It's much better now!

**Review by mrwrob — APPROVED** [review]
It's much better now :+1:

**Pull request merged**